### PR TITLE
Copy over formats module and use it on the feed page

### PR DESCRIPTION
**Pull request by uxengine**
Feed page:
bildschirmfoto 2017-07-20 um 10 11 44

### CODE REVIEW

File: pages/feed.js [hunk 1]
```diff
@@ -29,13 +66,7 @@ const Feed = graphql(
...
         return (
           <div>
-            {allArticles.map(article =>
-              <div key={article.id}>
-                <Link route="article" params={{ slug: article.slug }}>
-                  {article.title}
-                </Link>
-              </div>
-            )}
+            {allArticles.map(article => <Article article={article} />)}
```
Comment (tpreusse) [inline pages/feed.js]: `key` fehlt

File: pages/feed.js [hunk 2]
```diff
@@ -18,6 +37,24 @@ const allArticles = gql`
     }
   }
 `
+const Article = ({ article }) => {
+  const date = new Date(article.updatedAt)
+  return (
+    <article key={article.id} {...styles.article}>
```
Comment (tpreusse) [inline pages/feed.js]: no `key` needed here

File: pages/feed.js (hunk 1 above)
Comment (uxengine) [reply pages/feed.js]: Done

File: pages/feed.js (hunk 2 above)
Comment (uxengine) [reply pages/feed.js]: Done

**Comment by uxengine** [comment]
Comments addressed.

**Pull request merged**